heron/spi/src/java/com/twitter/heron/spi/utils/ReflectionUtils.java

```diff
@@ -28,6 +28,9 @@ private ReflectionUtils() {
   @SuppressWarnings("unchecked") // we don't know what T is until runtime
   public static <T> T newInstance(ClassLoader classLoader, String className)
       throws ClassNotFoundException, IllegalAccessException, InstantiationException {
+    if (className == null) {
+      throw new ClassNotFoundException("Can not instantiate class. className must not be null");
+    }
     return (T) classLoader.loadClass(className).newInstance();
```

Comment:
Thanks. I didn't review carefully here. @billonahill do you have any particular reason to add this check?